batch/test-locally.sh

```diff
@@ -1,5 +1,7 @@
 set -ex
 
+. activate hail-batch
```

Comment:
What does this do?

Reply:
[POSIX `. foo args ...`](http://pubs.opengroup.org/onlinepubs/009696699/utilities/dot.html) executes the script `foo` with the arguments `args ...` in the current environment, so if `foo` contains a line:
```bash
VARIABLE=value!
```
then the shell that called `. foo args` will have `VARIABLE` set to `value!`. `./foo` and `bash foo` do not have the same effect.

Practically speaking, it enables the hail-batch conda environment with all the binaries and packages described by `batch/environment.yml`. It's the POSIX-compliant way to say `source activate hail-batch`.